### PR TITLE
Adds better grid functionality

### PR DESCRIPTION
**Pull request by danleedham**
Another classic hola from 2017. To say how little I knew about the graphics then (this was all pre-Elections monsters) I seemed to actually manage to get a few little things to werk. 

screen shot 2018-04-10 at 21 26 24

### CODE REVIEW

File: public/admin/templates/grid.tmpl.html [hunk 1]
```diff
@@ -91,4 +127,12 @@ <h2>Grid Rows</h2>
     	allowEmpty:true,
     	showButtons: false
     });
+    
+    function showColorOptions() {
```
Comment (moschopsuk) [inline public/admin/templates/grid.tmpl.html]: Can these be Angular(ished) ?

**Review by moschopsuk — CHANGES_REQUESTED** [review]

File: public/admin/templates/grid.tmpl.html (hunk 1 above)
Comment (stephenrob) [reply public/admin/templates/grid.tmpl.html]: Agree using the angular way of updating this is better than by directly modifying the DOM

**Comment by benjaminkay93** [comment]
@moschopsuk I have picked it up and sorted it 👍

File: public/js/dashboard.js [hunk 2]
```diff
@@ -1124,7 +1132,14 @@ app.controller('tennisCGController', ['$scope', 'socket',
         function winGame(player) {
             // given the scoring player, get their opponent
             var opponent = (player == 1 ? 2 : 1);
-			
+<<<<<<< HEAD
```
Comment (moschopsuk) [inline public/js/dashboard.js]: erm

**Review by moschopsuk — CHANGES_REQUESTED** [review]

File: public/js/dashboard.js (hunk 2 above)
Comment (benjaminkay93) [reply public/js/dashboard.js]: how did that even work :')

**Review by moschopsuk — APPROVED** [review]

**Pull request merged**